### PR TITLE
IDateTimeFactory add parameter $time

### PR DESCRIPTION
**Pull request by MartkCz**

### CODE REVIEW

File: src/DateTime.php [hunk 1]
```diff
@@ -3,11 +3,17 @@
...
 use DateTimeInterface;
+use DateTimeZone;
 use Nette\Utils\DateTime as NetteDateTime;
 
 class DateTime extends NetteDateTime
 {
 
+	public function __construct(string $time = 'now', ?DateTimeZone $timezone = null)
+	{
+		parent::__construct($time, $timezone);
```
Comment (MartkCz) [inline src/DateTime.php:14]: Because of nette/di: is_a() expects parameter 2 to be string, null given

Comment (f3l1x) [reply src/DateTime.php:14]: Well, I don't get it. :-) What kind of problem does it solve?

Comment (f3l1x) [reply src/DateTime.php:14]: It's solved in nette/di (https://github.com/nette/di/issues/186). Does it still required?

Comment (MartkCz) [reply src/DateTime.php:14]: Yes, the only thing changed is error message to "Service 'xxx' (type of Contributte\Utils\IDateTimeFactory): Type hint for $time in create() doesn't match type hint in Contributte\Utils\DateTime constructor"

Comment (f3l1x) [reply src/DateTime.php:14]: I see. Thanks.

**Pull request merged**